db/migrate/20241002110620_create_decisions.rb

```diff
@@ -0,0 +1,16 @@
+class CreateDecisions < ActiveRecord::Migration[7.1]
+  def change
+    create_table :decisions, id: :uuid do |t|
+      t.references :crime_application, type: :uuid, foreign_key: true, null: true, index: { unique: true }
```

Comment:
Why is crime_application `null: true` ?

Reply:
Copy and paste error :sweat_smile: my bad!